docs/source/installation.md

```diff
@@ -20,6 +20,24 @@ cd transformers
 pip install .
 ```
 
+## Caching models
+
+This library provides pretrained models that will be downloaded and cached locally. Unless you specify a location with
+`cache_dir=...` when you use the `from_pretrained` method, these models will automatically be downloaded in the 
+folder given by the shell environment variable ``TRANSFORMERS_CACHE``. The default value for it will be the PyTorch 
+cache home followed by ``/transformers/`` (even if you don't have PyTorch installed). This is (by order of priority):
+
+  * shell environment variable ``ENV_TORCH_HOME``
+  * shell environment variable ``ENV_XDG_CACHE_HOME`` + ``/torch/``
+  * default: ``~/.cache/torch/``
+
+So if you don't have any specific environment variable set, the cache directory will be at 
+``~/.cache/torch/transformers/``.
+
+**Note:** If you have set a shell enviromnent variable for one of the predecessors of this library 
```

Comment:
Would be good to note that these variables are only read when the library is IMPORTED. I just spent quite a while figuring out why my ENV Variables were not being picked up and this was the reason why.

Reply:
How would you test your environment variables are picked without importing the library?

Reply:
I was importing the library and _then_ adding environment variables to modify the cache directory.

`
from transformers import pipeline
os.environ['TRANSFORMERS_CACHE'] = "/new/cache/dir"
`